src/maingoblint.ml

```diff
@@ -431,6 +432,7 @@ let main =
         Stats.reset Stats.SoftwareTimer;
         parse_arguments ();
         check_arguments ();
+        AfterConfig.run ();
 
         (* Cil.lowerConstants assumes wrap-around behavior for signed intger types, which conflicts with checking
           for overflows, as this will replace potential overflows with constants after wrap-around *)
```

Comment:
Maybe we should warn here if we change the settings?
Not really related to this PR, just saw this now

Reply:
And probably move to `check_arguments` too